### DISCUSSION TITLE
Contributors conference call #2: Early July

### DISCUSSION THREAD

**Discussion by ethomson** [libgit2/libgit2#5880] [General]
This issue is for planning the next contributors conference call in Early July, either week 28 (July 9-13) or week 29 (July 16-20).

I'd propose later in the week of week 29 (say Thursday 19 July).  I'll be at OSCON that week, so open source will be top of mind!  Plus I'll be jetlagged, so I'll be up early anyway and might as well do something useful.  😛

**Comment by carlosmn** [comment]
That day works for me.

**Comment by pks-t** [comment]
That date should work for me.
